aten/src/ATen/native/cuda/ScatterGatherKernel.cu

```diff
@@ -458,6 +416,8 @@ void scatter_reduce_cuda_kernel(Tensor& self, const int64_t dim, const Tensor& i
     cuda_scatter_gather_base_kernel<true, false>()(self, dim, index, src,
                                        "scatter_reduce_cuda_multiply_", reduce_multiply);
     break;
+  default:
+    TORCH_CHECK(false, "reduce argument must be either add or multiply.");
```

Comment:
Same as in the cpu kernel: I don't think we need this check, since it's already done in the META function.